src/hotspot/share/utilities/vmError.cpp

```diff
@@ -842,9 +845,20 @@ void VMError::report(outputStream* st, bool _verbose) {
        frame fr = _context ? os::fetch_frame_from_context(_context)
                            : os::current_frame();
 
-       print_native_stack(st, fr, _thread, buf, sizeof(buf));
+       print_native_stack(st, fr, _thread, true, buf, sizeof(buf));
        _print_native_stack_used = true;
      }
+     print_native_stack_succeeded = true;
+   }
+
+  STEP("retry printing native stack (no source info)")
```

Comment:
Should we move STEP under if (print_native_stack_succeeded) ?

Reply:
Hi Dmitry,

Thanks for the approval!

> Should we move STEP under if (print_native_stack_succeeded) ?

Wouldn't work, since STEP is defined as closing-bracket-condition-open-bracket, so it has to be at the first indentation level.